package.json

```diff
@@ -6,6 +6,7 @@
     "@emotion/react": "^11.4.1",
     "@emotion/styled": "^11.3.0",
     "@material-ui/core": "^4.12.3",
+    "@material-ui/styles": "^4.11.4",
```

Comment:
I think you might add the deprecated package

Reply:
I will work to remove this package.